Nodejs/MigrateToJsps/Jsps/LaunchJsonWriter.cs

```diff
@@ -89,13 +89,24 @@ public static void CreateLaunchJson(string projectDir, NjsprojFileModel njsprojF
                 var chromeLaunch = ChromeLaunchTemplate;
                 chromeLaunch.Url = chromeLaunch.Url.Replace("*PORT*", port);
 
-                Configuration[] launchConfigs = new Configuration[]
+                Configuration[] launchConfigs;
+
+                if (bool.Parse(njsprojFileModel.StartWebBrowser))
```

Comment:
As far as I understand, the ordering here is not going to achieve what you're looking for. When StartWebBrowser is set, we want VS to start both the browser and node  (i.e. use the compound config) and when it's not set it should just start node. Since VS will prioritize single configs, the StartWebBrowser=true scenario requires the user to select the compound config from the dropdown regardless of ordering. If we want to provide a better experience, we can put the compound config in the user file so that it will be selected by default in the dropdown when the migration finishes.